assets/js/UserProfile.jsx

```diff
@@ -37,6 +37,10 @@ class UserProfile extends React.Component {
     api.send_friend_request(this.user_id, this.fetchUser.bind(this));
   }
 
+  pokeUser() {
+      api.poke_user(this.user_id, this.fetchUser.bind(this));
```

Comment:
don't think `fetchUser` call is really needed, since the only change that happens is the points for the person who is doing the poking, not the person whose page we are on / receiving the poke

Reply:
though, it might be nice to show in the UI in some way that the poke was successful